src/Components/App/App.js

```diff
@@ -102,6 +96,7 @@ const App = () => {
       });
       setLoading(false)
       setVendors(updatedVendors);
+      setStoredVendors(updatedVendors)
```

Comment:
If anyone is wondeirng this is the fix we needed to eradicate the bug, we had to make the full circle of stored data basically